browser/brave_wallet/brave_wallet_p3a_unittest.cc

```diff
@@ -105,6 +107,16 @@ TEST_F(BraveWalletP3AUnitTest, DefaultSolanaWalletSetting) {
 }
 
 TEST_F(BraveWalletP3AUnitTest, KeyringCreated) {
+  base::test::ScopedFeatureList feature_list;
+  base::FieldTrialParams parameters;
+  parameters["create-default-solana-account"] = "false";
```

Comment:
can we use `brave_wallet::features::kCreateDefaultSolanaAccount.name`?

Reply:
Fixed